corporate/tests/test_stripe.py

```diff
@@ -7747,8 +7747,7 @@ def test_upgrade_user_to_fixed_price_monthly_basic_plan(self, *mocks: Mock) -> N
             self.assert_in_response(substring, response)
 
     @responses.activate
-    @mock_stripe()
-    def test_delete_configured_fixed_price_plan_offer(self, *mocks: Mock) -> None:
+    def test_delete_configured_fixed_price_plan_offer_no_active_plan(self) -> None:
```

Comment:
This test, and the new one, do not need stripe fixtures as all the tested actions are done in our database, so removed that decorator. This is helpful for knowing when modifying a test will require generating new fixtures.